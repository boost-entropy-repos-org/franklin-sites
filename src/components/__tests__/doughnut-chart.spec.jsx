import React from 'react';
import renderer from 'react-test-renderer';
import DoughnutChart from '../doughnut-chart';

describe('DoughnutChart component', () => {
  test('should render', () => {
    const component = renderer
      .create(
        <DoughnutChart percent={40}>
          <span>Some content</span>
        </DoughnutChart>,
      )
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});
