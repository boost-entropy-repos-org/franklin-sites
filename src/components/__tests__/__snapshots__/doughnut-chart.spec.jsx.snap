// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoughnutChart component should render 1`] = `
<span
  className="doughnut-chart--medium colour-azureish-white"
>
  <span
    className="doughnut-chart--medium__left-wrap"
  >
    <span
      className="doughnut-chart--medium__left-wrap__loader colour-sea-blue"
      style={
        Object {
          "transform": "rotate(0deg)",
        }
      }
    />
  </span>
  <span
    className="doughnut-chart--medium__right-wrap"
  >
    <span
      className="doughnut-chart--medium__right-wrap__loader colour-sea-blue"
      style={
        Object {
          "transform": "rotate(144deg)",
        }
      }
    />
  </span>
  <span
    className="doughnut-chart--medium__inner-circle"
    style={Object {}}
  >
    <span>
      Some content
    </span>
  </span>
</span>
`;
